src/esp/bindings/GfxBindings.cpp

```diff
@@ -151,6 +151,11 @@ void initGfxBindings(py::module& m) {
       .value("GLOBAL", LightPositionModel::GLOBAL)
       .value("OBJECT", LightPositionModel::OBJECT);
 
+  py::enum_<LightType>(
```

Comment:
We really should have a way to extract these lights types for the GLBs. For both importing lit scenes as well as importing objects like lamps.

Reply:
I agree.  Currently this is only to facilitate the creation of LightInfos appropriately from LightInstanceAttributes, but I think eventually the LightInfos would benefit from having the desired type more explicitly specified.